addons/web_editor/static/lib/odoo-editor/src/commands/commands.js

```diff
@@ -584,6 +584,13 @@ export const editorCommands = {
         }
 
         let target = [...(blocks.size ? blocks : li)];
+        if (blocks.size) {
+            for (const block of blocks) {
+                if (block.style) {
+                    block.style.padding = null;
+                }
+            }
+        }
```

Comment:
I think your new block of code is specific enough to a particular case to warrant a coment.